config/env.go

```diff
@@ -7,6 +7,10 @@ import (
 )
 
 func expandEnvironment(nodes []Node) []Node {
+	if nodes == nil {
```

Comment:
Is there a reason why a `len` check isn't used here?

Reply:
Added a comment to explain.